apps/web/src/lang/fr.json

```
@@ -2667,6 +2667,10 @@
     "defaultMessage": "Non démontré suffisamment dans le cadre de la présente méthode d’évaluation – faire passer à de plus amples mises à l’essai. (Ne s’applique qu’aux occasions où les points sont cumulatifs à l’échelle des méthodes d’évaluation. Ne s’applique pas dans les cas où les candidats doivent répondre à certains critères essentiels à chaque étape).",
     "description": "Note for when an assessment was unsuccessful but put on hold"
   },
+  "DOn3Hm": {
+    "defaultMessage": "Adresse courriel non valide. Essayez de nouveau ou envoyez un courriel, o <anchorTag>{emailAddress}</anchorTag>.",
```

Based on the English equivalent, I think this is a typo.

```suggestion
    "defaultMessage": "Adresse courriel non valide. Essayez de nouveau ou envoyez un courriel à <anchorTag>{emailAddress}</anchorTag>.",
```

---

Thanks!  Fix checks out on deepl.